python/MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+# C++ files
+recursive-include rmm *.hpp
+
+# Cython files
+recursive-include rmm *.pxd
```

Comment:
We need to include `*.pyx, *.pxi` as well. And we need to include CMake files so that source distributions (if created) can properly use scikit-build.

Maybe test locally that you can create a source distribution and build from it. We don't need to do that in CI since we don't deploy that way, but it helps us know our MANIFEST is right.

I maintained another scikit-build + Cython package in grad school and this is how it was configured, for reference: https://github.com/glotzerlab/freud/blob/master/MANIFEST.in

Reply:
Both of these came from what was already set in `setup.py`

https://github.com/rapidsai/rmm/blob/637d5fd37643c4d783da1ed1de62ba2102dd290e/python/setup.py#L31

Did start playing with adding other things initially, but backed out as it felt like scope creep

Reply:
I went ahead and made the minimal set of changes required for a working sdist. I was OK considering that in scope for this PR.